Makefile

```diff
@@ -201,3 +204,14 @@ local-a4:
 		$(VIRTUAL_ENV)/bin/python manage.py migrate; \
 		npm link ../adhocracy4; \
 	fi
+
+.PHONY: celery-worker-start
+celery-worker-start:
+	$(VIRTUAL_ENV)/bin/celery --app adhocracy-plus worker
```

Comment:
good to add `--loglevel INFO` after the worker

Reply:
agreed, done.